skrub/tests/test_fuzzy_join.py

```diff
@@ -59,80 +49,79 @@ def test_fuzzy_join(px, analyzer: Literal["char", "char_wb", "word"]):
         add_match_info=True,
     )
     # Joining is always done on the left table and thus takes it shape:
-    assert df_joined2.shape == (len(df2), n_cols)
+    assert ns.shape(df_joined2) == (len(df2), n_cols)
 
-    df1["a2"] = 1
+    ns.with_columns(df1, "a2", 1)
```

Comment:
if this is the only place you need `with_columns` maybe we can add a TODO and keep `df1["a2"] = 1`, and deal with it in another PR to avoid blocking this one, as `with_columns` might be slighly more tricky than it would seem at first

Reply:
Done :)